site/docs/components/flex-layout/examples.mdx

```diff
@@ -0,0 +1,86 @@
+
+<LivePreview componentName="flex-layout" exampleName="Default">
+
+### Default
+
+By default, Flex Layout displays items along the horizontal axis in a row, aligned to the start of the container. The spacing between items is driven by the property `gap`, which is a multiple of the [Salt base unit](/salt/foundations/spacing) and is set to 3. The dimensions of the layout (height or width) are driven by the items contained within it.
+
+#### Guidance
+
+Set the `wrap` prop to `true` to overflow items onto subsequent rows when they no longer fit in the container.
+
+</LivePreview>
+
+<LivePreview componentName="flex-layout" exampleName="ColumnDirection">
+
+### Column direction
+
+The Flex Layout `direction` prop defines the main axis that items are displayed along. Set `direction` to `column` to display items in a vertical direction.
+
+</LivePreview>
+
+<LivePreview componentName="flex-layout" exampleName="LayoutPosition">
+
+### Layout position
+
+Control the position of Flex Layout and its items using the `align` and `justify` props.
+
+Use `justify` to define how to distribute any remaining space between or around all of the items along the main-axis of Flex Layout (i.e., along the row or column).
+
+If there is additional space available in the cross-axis of the layout (i.e., beside a row, or above/below a column), the `align` prop defines how items should be positioned within that space.
+
+</LivePreview>
+
+<LivePreview componentName="flex-layout" exampleName="ResponsiveLayout">
+
+### Responsive layout
+
+Flex Layout supports responsive props, so you can define different behaviors based on viewport size (e.g., switching `direction` with screen size). You can use the default [Salt breakpoints](/salt/foundations/responsiveness#breakpoints) or pass in your own using `SaltProvider`.
+
+#### What are responsive props?
+
+A responsive prop is a prop that takes either multiple values (one value per breakpoint), or a single value that would take effect across all different screen sizes. For example, a layout that will responsively change direction from row to column at specific breakpoints or viewport sizes:
+
+```js
+direction={{ xs: "row", sm: "column", lg: "row", md: "column", xl: "row" }}
+```
+
+</LivePreview>
+
+<LivePreview componentName="flex-layout" exampleName="FlexItemPosition">
+
+### Flex item position
+
+Use the Flex Item `align` prop to set an alignment for a specific item in the layout. Use the `align` prop to set the item’s position along the cross-axis of the layout (i.e., perpendicular to the `direction`–`row` or `column`).
+
+</LivePreview>
+
+<LivePreview componentName="flex-layout" exampleName="FlexItemSize">
+
+### Flex item size
+
+The Flex Item `grow` prop enables specific items in the layout to grow and fill any remaining space that is available in the layout:
+
+- By default, `grow` is set to 0 and items will be displayed in their ‘normal’ size.
+- If an item is set to ‘1’ it will fill the available space.
```

Comment:
use of number is different here v.s. guidance, quote or backtick?

Reply:
@sion-docs which one should we use?